Online/Menus/CSS/TextEntryScreen/OnConfirmButtonAPress.asm

```diff
@@ -9,6 +9,22 @@ lbz r3, OFST_R13_NAME_ENTRY_MODE(r13)
 cmpwi r3, 0
 beq EXIT
 
+# Check to see if there's an autcomplete suggestion that hasn't 
```

Comment:
So my current solve for this, rather than making a separate function, is that I did something similar to what you'd recommended with the LRInputHandler. I noticed that most of the behavior in these two files were similar, with the exception of where they ultimately branched to. So I now have the OnConfirmButtonAPress code pass an argument, and then branch to a shared OnConfirmButtonHandler. 

I was thinking of making a general function for clearing the text, like you'd suggested, but it seems unlikely that logic would be helpful anywhere else. Additionally, it seems like there were only one or two unused addresses left in Common.s.